tests/converters/test_base_converter.py

```diff
@@ -0,0 +1,244 @@
+    then the value of the key word argument should be none
+    """
+    link = Link(
+        [KwargPath("not_existing_value", ["foo", "bar", "baz", "qux"])],
+        lambda x, not_existing_value: (x, not_existing_value),
+    )
+    assert link.get("qux_value", DICTIONNARY) == ("qux_value", None)
+
+
+@pytest.mark.parametrize(
+    "path_field,path,expected_path",
+    [
+        ("", ["foo"], ["foo"]),
+        ("", ["foo", "bar"], ["foo", "bar"]),
+        ("", lambda x: x, ""),
+        ("some value", lambda x: x, "some value"),
+        (
+            "some value",
+            Link(
+                [KwargPath("bar_value", ["foo", "bar"])],
+                lambda x, bar_value: [x, bar_value],
+            ),
+            ["some value", "baz"],
+        ),
+    ],
+)
+@pytest.mark.parametrize(
+    "value_field,value_arg,expected_value",
+    [
+        ("", ["static_value"], "static_value"),
+        ("", [123], 123),
+        ("", [{"foo": "bar"}], {"foo": "bar"}),
+        ("", [], ""),
+        ("some value", [], "some value"),
+        (
+            "some value",
+            [
+                Link(
+                    [KwargPath("bar_value", ["foo", "bar"])],
+                    lambda x, bar_value: (x, bar_value),
+                )
+            ],
+            ("some value", "baz"),
+        ),
+    ],
+)  # pylint: disable=too-many-arguments
+def test_go_to(path_field, path, expected_path, value_field, value_arg, expected_value):
+    """Check that for the specified kwargs, value we get the expected path and value"""
+    args = [path] + value_arg
+    go_to = GoTo(*args)
+    assert go_to.path(path_field, DICTIONNARY) == expected_path
+    assert go_to.value(value_field, DICTIONNARY) == expected_value
+
+
+def test_convert_without_nesting():
+    """Check the convert function for a schmea without nested fields"""
+    converter = BaseConverter()
+    assert converter.convert({}) == {}
+
+    converter._schema = ContextModuleSchema()  # pylint: disable=protected-access
+    context_module = {}
+
+    # When the event dict (context_module) is not valid (raises ValidationError) we return None
+    assert converter.convert(context_module) is None
+    context_module["display_name"] = "display_name_value"
+    assert converter.convert(context_module) is None
+
+    # When the _schema contains more fields than we have defined, Attribute error is raised
+    context_module[
+        "usage_key"
+    ] = "usage_key_value"  # Now context_module is a valid event
+    with pytest.raises(
+        AttributeError,
+        match="'BaseConverter' object has no attribute '(display_name|usage_key)'",
+    ):
+        converter.convert(context_module)
+
+    converter.display_name = GoTo(["foo"])
+    # Now only usage_key is missing
+    with pytest.raises(
+        AttributeError, match="'BaseConverter' object has no attribute 'usage_key'"
+    ):
+        converter.convert(context_module)
+    converter.usage_key = GoTo(["bar"])
```

Comment:
Some suggestions to change the name of the GoTo class

```suggestion
    converter.usage_key = MoveTo(["bar"])
```
```suggestion
    converter.usage_key = At(["bar"])
```
```suggestion
    converter.usage_key = FieldAt(["bar"])
```


```suggestion
    converter.usage_key = PathValue(["bar"])
```